docs/ml-features.md

```diff
@@ -53,7 +53,24 @@ Refer to the [HashingTF Java docs](api/java/org/apache/spark/ml/feature/HashingT
 Refer to the [HashingTF Python docs](api/python/pyspark.ml.html#pyspark.ml.feature.HashingTF) and
 the [IDF Python docs](api/python/pyspark.ml.html#pyspark.ml.feature.IDF) for more details on the API.
 
-{% include_example python/ml/tf_idf_example.py %}
```

Comment:
Here do not change the example imported by `include_example` to raw code back. The JIRA issue aims to change all raw code snippts with `include_example`.